otter/tap/api.py

```diff
@@ -227,6 +228,17 @@ def makeService(config):
         admin_service = service(str(admin_port), admin_site)
         admin_service.setServiceParent(s)
 
+    # setup cloud feed
+    cf_conf = config.get('cloudfeeds', None)
+    if cf_conf is not None:
+        service_configs[ServiceType.CLOUD_FEEDS] = {
```

Comment:
It can be optional even in get_service_config -- it could have this same conditional logic to only include CLOUD_FEEDS if it's in the config file.